apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/sdl/GraphSdlSchema.kt

```diff
@@ -1,6 +1,12 @@
 package com.apollographql.apollo.compiler.parser.sdl
 
-data class GraphSdlSchema(
+import com.apollographql.apollo.compiler.ir.SourceLocation
+import com.apollographql.apollo.compiler.parser.sdl.GraphSDLSchemaParser.parse
+import java.io.File
+import java.util.Locale
+import com.apollographql.apollo.compiler.parser.Schema as IntrospectionSchema
```

Comment:
Could we rename `Schema` to `IntropspectionSchema` instead to avoid the aliased import?

Reply:
For sure, it's in my list for the next PR